server/db/models/Product.js

```diff
@@ -25,18 +26,16 @@ const Product = db.define('product', {
   ISBN13: {
     type: Sequelize.STRING,
     allowNull: false,
-    unique: true,
   },
   imageUrl: {
-    type: Sequelize.STRING,
-    //defaultValue: find default photo for public folder
+    type: Sequelize.TEXT,
   },
   price: {
     type: Sequelize.INTEGER,
     allowNull: false,
   },
   length: {
-    type: Sequelize.STRING,
+    type: Sequelize.INTEGER,
```

Comment:
I think we had made this string because we thought it would be formatted like length: "300 pages" or length: "2 hours" in the case of an audiobook.  But I guess since we're not selling audiobooks, this just assumes pages? 

That makes sense.  Should we refactor the front-end components to say "pages" after the {length} then so it's a little nicer UI experience?

Reply:
agree